### DISCUSSION TITLE
Build Status

### DISCUSSION THREAD

**Discussion by mwilson-atlassian** [scientistproject/Scientist.net#158] [General]
I noticed that the build widget on the `README.md` page indicates that the build is failing. I checked the AppVeyor logs and it looks like there might have been a transient network issue. 

```
Attempting to gather dependencies information for package 'FAKE.Core.4.4.2' with respect to project 'tools', targeting 'Any,Version=v0.0'
The underlying connection was closed: Could not establish trust relationship for the SSL/TLS secure channel.
The system cannot find the path specified.
Command exited with code 3
```

Can we give the build a manual run and see if it's satisfied now?

<img width="520" alt="image" src="https://github.com/scientistproject/Scientist.net/assets/39309948/fce649fd-c9ae-4690-bb60-fe304d45aa2d">

**Comment by JoshHiles** [comment]
Not sure who holds the keys to the AppVeyor however i have been doing a bit of work in background on using GitHub Actions to deploy new releases to NuGet

**Reply by mwilson-atlassian** [reply]
Gotcha. So the idea would be to replace AppVeyor with github actions? Looks like there are windows runners that support .net 4: https://github.com/orgs/community/discussions/25253#discussioncomment-3247123

**Reply by JoshHiles** [reply]
I think so, makes more sense for everything to be in one place!
In terms on the runners / implementation, I'm looking at doing it on the 3.0.0 branch with the new solution etc.
I need to ask for the NuGet token so we can start pre releases!

**Reply by JoshHiles** [reply]
Hopefully you can see this https://github.com/JoshHiles/Scientist.net/pull/2, just to show what im thinking going forward
